Source/FieldSolver/SpectralSolver/SpectralKSpace.cpp

```diff
@@ -9,6 +9,7 @@
 #include "SpectralKSpace.H"
 
 #include <cmath>
+#include <math.h>
```

Comment:
Is this change really needed here?

Reply:
Was adding it temporarily, but forgot to remove it in previous cleanups. Fixed in commit df1091def0048e75e7a86.